core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -1031,20 +1030,46 @@ class HttpGoogleServicesDAO(
     }
   }
 
+  override def addGoogleProjectLabel(googleProject: GoogleProjectId, key: String, value: String): Future[Unit] = {
```

Comment:
do we do any string validation for key, value? some prior code example we did in DM: https://github.com/broadinstitute/gcp-dm-templates/blob/master/firecloud_project.py#L371

Reply:
added validation ; waiting on answers from PO about how this should look https://broadworkbench.atlassian.net/browse/CA-1224